EIPS/eip-131.md

```diff
@@ -0,0 +1,81 @@
+---
+eip: 131
+title: Add precompiled BLAKE2b contract
+author: Tjaden Hess (@tjade273), Jay Graber, James Hancock (@madeoftin)
+discussions-to: https://gitter.im/EIPshepherding/eip-131
```

Comment:
I hope this is my last comment.

The original issue (https://github.com/ethereum/EIPs/issues/152) has quite a lot of discussion. If nothing else, it would be nice to include it in a `References` section.

However @Souptacular posted there and on the previous PR the link to FEM (https://ethereum-magicians.org/t/blake2b-f-precompile/3157) as the discussion URL.

Perhaps because of that it would make sense using it for `discussions-to` ?

Reply:
yeah, that does make sense. Thanks for the review, I will get to work on the rest of the comments. I had a developer step up to work on this. So that is good news.